packages/integrations/image/src/lib/get-picture.ts

```diff
@@ -51,7 +51,7 @@ export async function getPicture(params: GetPictureParams): Promise<GetPictureRe
 	}
 
 	if (!widths || !Array.isArray(widths)) {
-		throw new Error('[@astrojs/image] at least one `width` is required');
+		throw new Error('[@astrojs/image] at least one `width` is required. ex: `widths={[100]}`');
```

Comment:
Maybe I should put this in a separate PR 😓 

I've been confused every time I've gotten this error, trying to pass `width="100vw"` then `widths="100vw"` until I get to `widths=[100]` and then `widths={[100]}` 😅 

weird that astro devtools doesn't seem to pick it up as an error

Reply:
Yeah, this message isn't great. I'm fine with this being in the same PR, it's just a very simple string change.